### PR TITLE
ticdc: add ga version, remove experiment warning

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

TiCDC will GA since v4.0.5

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/4353 https://github.com/pingcap/docs-cn/pull/4364
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: whats-new-in-tidb-4.0.md [hunk 1]
```diff
@@ -69,22 +67,26 @@ TiUP is a new package manager tool introduced in v4.0 that is used to manage all
...
 + The Follower Read feature refers to using any follower replica of a Region to serve a read request under the premise of strongly consistent reads. This feature improves the throughput of the TiDB cluster and reduces the load of the leader. It contains a series of load balancing mechanisms that offload TiKV read loads from the leader replica to the follower replica in a Region. TiKV's Follower Read implementation guarantees the consistency of data reading; combined with Snapshot Isolation in TiDB, this implementation provides users with strongly consistent reads. See [Follower Read](/follower-read.md) for details.
+
+## TiCDC
+
+TiCDC is a tool for replicating the incremental data of TiDB. This tool is implemented by pulling TiKV change logs, which ensures high reliability and availability of data. You can subscribe to the change information of data, and the system automatically sends data to the downstream. Currently, the downstream database must be MySQL compatible (such as MySQL and TiDB) or Kafka. You can also extend the supported downstream systems based on the [TiCDC Open Protocol](/ticdc/ticdc-open-protocol.md). See [TiCDC](/ticdc/ticdc-overview.md) for details.
```
Comment (amyangfei) [inline whats-new-in-tidb-4.0.md:92]: Oh, we support MySQL compatible database, Kafka and Pulsar now. We need to update the Chinese document too.

**Review by ran-huang — COMMENTED** [review]
LGTM

File: whats-new-in-tidb-4.0.md (hunk 1 above)
Comment (TomShawn) [inline whats-new-in-tidb-4.0.md:92]:
```suggestion
TiCDC is a tool for replicating the incremental data of TiDB. This tool is implemented by pulling TiKV change logs, which ensures high reliability and availability of data. You can subscribe to the change information of data, and the system automatically sends data to the downstream. Currently, the downstream database must be MySQL compatible (such as MySQL and TiDB) or Kafka and Pulsar. You can also extend the supported downstream systems based on the [TiCDC Open Protocol](/ticdc/ticdc-open-protocol.md). See [TiCDC](/ticdc/ticdc-overview.md) for details.
```

Comment (TomShawn) [reply whats-new-in-tidb-4.0.md]: Updated in https://github.com/pingcap/docs/pull/3754/commits/6107168f0205494b36b44279dd210852b5fdb726

**Review by amyangfei — COMMENTED** [review]
LGTM

**Review by ran-huang — APPROVED** [review]

**Pull request merged**